### PR TITLE
Registration process rewording

### PR DESCRIPTION
**Pull request by RvanB**

### CODE REVIEW

File: client/components/public/FAQ.jsx [hunk 1]
```diff
@@ -382,7 +382,7 @@ FAQ = class FAQ extends Component {
         </Accordion>
 
         <br/>
-        <p>Last Updated: January 2019</p>
+        <p>Last Updated: January 2020</p>
```
Comment (cabeese) [inline client/components/public/FAQ.jsx]: Isn't it November 2019 right now? ;)

**Review by cabeese — CHANGES_REQUESTED** [review]
Changes here look good. I think we still need to update the instructions in the verification email to match the new phrasing we're adopting this year. Check out `lib/imports/emails.js` and `lib/imports/sendTickets.js` (I think only the former of the two will need to be changed).

**Comment by cabeese** [comment]
While you're at it, would you fix the typo on `emails.js:14` where "invitation" is misspelled "invitiation"? I don't want to fix it myself and create merge conflicts. TY!

File: lib/imports/emails.js [hunk 2]
```diff
@@ -32,19 +32,24 @@ ${signature}
 
 export const registrationInfoHTML = `
 <p>
-  <h4>Registration for the ${eventYear} ${siteName} is a 3-step process:</h4>
+  <h4>Registration for the ${eventYear} ${siteName} is a 4-step process:</h4>
   <ol>
     <li>
       Create account at <a target="_blank" href="${siteURL}/register">${siteURL}/register</a>.<br/>
+      Verify your email before continuing.
```
Comment (cabeese) [inline lib/imports/emails.js]: Could you add a `<br />` at the end of this line?

File: lib/imports/emails.js [hunk 3]
```diff
@@ -32,19 +32,24 @@ ${signature}
...
       Deadline: <strong>11:59 PM ${registrationCloseDate}</strong>
     </li>
     <li>
-      Purchase ticket code at <a target="_blank" href="https://commerce.cashnet.com/TheGreatPuzzleHunt2019">https://commerce.cashnet.com/TheGreatPuzzleHunt2019</a>.<br/>
-      Deadline: <strong>10:00 AM ${eventDate}</strong>
+      Purchase ticket code(s) at <a target="_blank" href="https://commerce.cashnet.com/TheGreatPuzzleHunt2020">https://commerce.cashnet.com/TheGreatPuzzleHunt2020</a>.<br/>
+      Ticket code(s) will be sent to your email.
```
Comment (cabeese) [inline lib/imports/emails.js]: `<br />` at end

File: lib/imports/emails.js [hunk 4]
```diff
@@ -32,19 +32,24 @@ ${signature}
...
+      Deadline: <strong>9:30 AM ${eventDate}</strong>
     </li>
     <li>
-      Redeem ticket code at <a target="_blank" href="${siteURL}/redeem">${siteURL}/redeem</a>.<br/>
-      Deadline: <strong>10:00 AM ${eventDate}</strong>
+      Redeem ticket code(s) at <a target="_blank" href="${siteURL}/redeem">${siteURL}/redeem</a>.<br/>
+      Deadline: <strong>9:30 AM ${eventDate}</strong>
+    </li>
+    <li>
+      Set-up/join team at <a target="_blank" href="http://greatpuzzlehunt.com/team">${siteURL}/team</a>.<br/>
```
Comment (cabeese) [inline lib/imports/emails.js]: Replace the URL with `${siteURL}` for consistency (also b/c https)

**Review by cabeese — CHANGES_REQUESTED** [review]

**Review by cabeese — APPROVED** [review]
Awesome, @RvanB. Were you able to send a test email and at least see its content in the console log to ensure the email was formatted as expected after your changes?

**Comment by RvanB** [comment]
Yeah I created a new account to see what the verification email looks like, looks good in the logs!

**Pull request merged**